evm/src/fuzz/strategies/param.rs

```diff
@@ -73,25 +73,26 @@ pub fn fuzz_param(param: &ParamType) -> impl Strategy<Value = Token> {
 /// fuzz state.
 ///
 /// Works with ABI Encoder v2 tuples.
-pub fn fuzz_param_from_state(param: &ParamType, state: EvmFuzzState) -> BoxedStrategy<Token> {
+pub fn fuzz_param_from_state(param: &ParamType, arc_state: EvmFuzzState) -> SBoxedStrategy<Token> {
```

Comment:
From my understanding, Send + Sync is necessary for the `Executor` since we use `par_iter()` for running normal and fuzz tests.

Since I added `RandomCallGenerator` to the `Executor`, it requires a strategy which is also `sboxed`. But instead of having all strategies be boxed, I should only have set that to `override_call_strat`. Pushed a fix.

Reply:
`EvmFuzzState` needs to be accessible from the executor too, to insert values from the `memory` and `stack`. Thus `Arc`.